uint32 isFaceDetected
float64 x
float64 y
---
bool face_detected
